### ISSUE TITLE
Ask: How to help convergence 

### ISSUE THREAD

**Issue by edmBernard** [zhreshold/mxnet-ssd#52]
I try to train a one classe detector. (my data set contain 19000 labeled images)
My training got rapidly stuck with :
```
INFO:root:Epoch[0] Batch [260]	Speed: 53.32 samples/sec	Train-Acc=0.750000
INFO:root:Epoch[0] Batch [260]	Speed: 53.32 samples/sec	Train-ObjectAcc=0.000000
INFO:root:Epoch[0] Batch [260]	Speed: 53.32 samples/sec	Train-SmoothL1=10.509061
```

I try to change learning rate, change batch size.
I also test to retrain from scratch instead of vgg16_reduced.
Have you tips to help convergence ?

**Comment by edmBernard** [comment]
The problem is not from tuning parameters :(
I still got this issues : 
https://github.com/zhreshold/mxnet-ssd/issues/51
I close as it duplicate.
